src/Executable.js

```diff
@@ -166,6 +188,18 @@ export default class Executable {
         return this;
     }
 
+    /**
+     * Set the node account IDs of the transaction
+     *
+     * @param {AccountId[]} nodeIds
+     * @returns {this}
+     */
+    setTransactionNodeIds(nodeIds) {
```

Comment:
I think this should not be exposed as an API

Reply:
You are correct; it’s better to limit this property to only this specific use case. Removed the getter/setters for this property.